nemo/collections/common/tokenizers/huggingface/auto_tokenizer.py

```diff
@@ -71,13 +71,13 @@ def __init__(
         self,
         pretrained_model_name: str,
         vocab_file: Optional[str] = None,
-        mask_token: Optional[str] = None,
-        bos_token: Optional[str] = None,
-        eos_token: Optional[str] = None,
-        pad_token: Optional[str] = None,
-        sep_token: Optional[str] = None,
-        cls_token: Optional[str] = None,
-        unk_token: Optional[str] = None,
+        mask_token: Optional[str] = '[MASK]',
```

Comment:
Why we set the default values for special tokens? Don't we want user to get error when she tried to use a special token when it is not set or exists for a specific model?

Reply:
no, we're setting defaults for preprocessing scripts to work, otherwise during preprocessing we won't bet an error, but 'None' will be added instead of token_ids. User always ca overwrite the defaults.